app/views/welcome_mailer/notify_newcomer.html.erb

```diff
@@ -18,6 +18,13 @@
 
     Looking forward to hearing from you soon,
 
+    Please click on the link below to verify your email.
+
+    <% verifaction_link="https://publiclab.org/verify/#{@user.generate_token}" %>
```

Comment:
Here we could replace `publiclab.org` with `<%= ActionMailer::Base.default_url_options[:host] %>` so it's not host-specific!

Reply:
Yeah I fixed that.